lib/auth/oidc.go

```diff
@@ -44,14 +43,7 @@ import (
 	"github.com/gravitational/trace"
 )
 
-const (
-	oidcNoRolesErrorMessage = "No roles mapped from claims. The mappings may contain typos."
-)
-
-// IsOIDCNoRolesError checks if an error results from a failure to map roles from claims.
-func IsOIDCNoRolesError(err error) bool {
-	return trace.IsAccessDenied(err) && strings.Contains(err.Error(), oidcNoRolesErrorMessage)
-}
+var OIDCNoRolesError = trace.AccessDenied("No roles mapped from claims. The mappings may contain typos.")
```

Comment:
Does this need to be exported? It's safer if we can leave it unexported - this way other packages can't reassign it to something else.